crds/roman/tpns/includes/nir_subarray.tpn

```diff
@@ -0,0 +1,44 @@
+# This TPN defines checks for the NIR-related SUBARRAY supporting keywords
```

Comment:
This is not needed in the beginning, for build 0. The common parts can be left in this PR or it can be removed altogether until we have more information about the subarrays in Roman.

Reply:
By "This", you mean the file?  So, I presume the I should delete the whole file, correct?

Reply:
Oh, Todd says below to delete the whole tpns directory.